lib/rubocop/cop/rspec/file_path.rb

```diff
@@ -130,6 +130,13 @@ def expected_path(constant)
         end
 
         def camel_to_snake_case(string)
+          if defined?(ActiveSupport::Inflector)
```

Comment:
There are other libraries that can be used to set inflectors: 
 - [dry-inflector](https://dry-rb.org/gems/dry-inflector/0.1/)
 - [zeitwerk](https://github.com/fxn/zeitwerk/blob/main/lib/zeitwerk/inflector.rb)

I would suggest providing a script field in the YAML files that could be run such that `String#underscore` could be defined if desired. 

```yaml
Rspec/FilePath:
  LoadInfector: |-
    require 'active_support/inflector'
    require './config/initializers/inflections'
```

Alternatively, we could add a field for the Inflector gem and/or definition file:

```yaml
Rspec/FilePath:
  UseInflectorGem: ActiveSupport
  InflectorDefinition: ./config/initializers/inflections.rb
```

Doing either of these should then allow `if string.respond_to?(:underscore)` to be used instead of hard-coding a library and a file location.

Reply:
Another alternative may simple be to make sure the file name matches the locations in which the class is defined. (e.g. if MyHTMLModule::XMLReader is defined in `app/services/my_html_module/xml_reader.rb` that this cop expects it spec to be in `spec/services/my_html_module/xml_reader_spec.rb` regardless of inflection rules.